Makefile

```diff
@@ -1,57 +0,0 @@
-.PHONY: clean-pyc clean-build docs clean
```

Comment:
In some ways, I like what is in this file more than the way we are doing it in the other repos.  Please don't remove.

(At least, not without some discussion and an explicit replacement for the functionality.  But just don't.  This is way outside of the stated scope for this ticket.)